README.md

```diff
@@ -47,15 +47,16 @@ In **PowerShell** or in a **Command Prompt** window:
     ```shell
     apt-get install \
         zlib1g-dev libssl-dev libncurses-dev libffi-dev \
-        libsqlite3-dev libreadline-dev libbz2-dev
+        libsqlite3-dev libreadline-dev libbz2-dev build-essential
     ```
 
   - On **Amazon Linux/Fedora**, run:
 
     ```shell
     yum install \
         zlib-devel openssl-devel ncurses-devel libffi-devel \
-        sqlite-devel.x86_64 readline-devel.x86_64 bzip2-devel.x86_64
+        sqlite-devel.x86_64 readline-devel.x86_64 bzip2-devel.x86_64 \
+    yum group install "Development Tools"
```

Comment:
Put this command before the previous one and eliminate the trailing `\` at the end of the previous command.